### ISSUE TITLE
Resolve VcfUtils FIXMEs

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#3296]
Resolve the following FIXMEs in org.molgenis.data.vcf.utils.VcfUtils:

```
// FIXME: This removes 'FLAG' fields? see http://samtools.github.io/hts-specs/VCFv4.2.pdf
if (vcfEntity.getString(attributeMetaData.getName()) != null) 
```

```
// FIXME: chance to clash with existing ID and NAME columns in FORMAT ?? what happens then?
if (!sampleAttribute.equals(VcfRepository.ID) && !sampleAttribute.equals(VcfRepository.NAME))
```

```
// FIXME: do we expect this??
if (firstSample && formatColumn.length() > 0)
```

```
// FIXME: once we support list of primitives we can calculate based on combination of type and nillable
sb.append(",Number=.");
```

**Comment by mswertz** [comment]
part of VCF cleanup story molgenis#3504

**Comment by dennishendriksen** [comment]
Fixed in MOLGENIS 1.21
